docs/React.md

```diff
@@ -100,7 +101,7 @@ Read [a manual](i18n.md).
 
 ## Testing
 
-Testing is done with `Jasmine` and [`Enzyme`](https://github.com/airbnb/enzyme).
+Testing is done with [`jest`](https://jestjs.io/en/) and [`React Testing Library`](https://testing-library.com/docs/react-testing-library/intro).
 
 TODO add an example when references are in master.
```

Comment:
> TODO add an example when references are in master.

Could be removed — I think the idea was to link to some examples. Maybe writing what's the typical path for test files would be enough?

Reply:
Yup, I added the path matcher and a link to a test.